couchdb/http.py

```diff
@@ -482,17 +483,20 @@ class ConnectionPool(object):
     def __init__(self, timeout, disable_ssl_verification=False):
         self.timeout = timeout
         self.disable_ssl_verification = disable_ssl_verification
-        self.conns = {} # HTTP connections keyed by (scheme, host)
+        self.conns = {} # HTTP connections keyed by (current_process_pid, scheme, host)
         self.lock = Lock()
 
-    def get(self, url):
+    @property
+    def _current_process_id(self):
```

Comment:
Not sure what this property buys us? Just using `os.getpid()` is actually shorter to write, and it's used in just two places.